notebook/pipeline/_notebook_op.py

```diff
@@ -60,15 +61,23 @@ def __init__(self,
         self.cos_dependencies_archive = cos_dependencies_archive
         self.container_work_dir = "jupyter-work-dir"
         self.bootstrap_script_url = bootstrap_script_url
+        self.requirements_url = requirements_url
         self.pipeline_outputs = pipeline_outputs
         self.pipeline_inputs = pipeline_inputs
 
         if self.bootstrap_script_url is None:
             """ If bootstrap_script arg with URL not provided, use the one baked in here.
             """
 
-            self.bootstrap_script_url = 'https://raw.githubusercontent.com/elyra-ai/' \
-                                        'kfp-notebook/master/etc/docker-scripts/bootstrapper.py'
+            self.bootstrap_script_url = 'https://raw.githubusercontent.com/akchinSTC/' \
+                                        'kfp-notebook/boot-version/etc/docker-scripts/bootstrapper.py'
+
+        if self.requirements_url is None:
+            """ If bootstrap_script arg with URL not provided, use the one baked in here.
+            """
```

Comment:
I think help-string comments are reserved for method and class signatures.  Not sure if there's a PEP for this kind of thing.  (Same comment for bootstrap_script_url block above).

Reply:
Removed both